migrate.py

```diff
@@ -101,12 +101,27 @@ def remove(path):
 def actually_remove(checkout_path, namespace, collection):
     global REMOVE
 
+    sanity_ignore = read_lines_from_file(os.path.join(checkout_path, 'test/sanity/ignore.txt'))
+    new_sanity_ignore = defaultdict(list)
+    for ignore in sanity_ignore:
+        values = ignore.split(' ')
```

Comment:
or better partition it so that variable names could carry some context
```suggestion
        file_path, sep, ignored_rules = ignore.partition(' ')
```